app/templates/road-sign-concepts/road-sign-concept.hbs

```diff
@@ -3,220 +3,435 @@
+          </s.menu>
+          <s.content as |c|>
+            <c.header>
+              <th class="data-table__header-title">
+                {{t "roadSignConcept.attr.code"}}
+              </th>
+              <th class="data-table__header-title">
+                {{t "roadSignConcept.attr.imageHeader"}}
+              </th>
+              <th class="data-table__header-title">
+                {{t "roadSignConcept.attr.meaning"}}
+              </th>
+              <th class="data-table__header-title">
+                {{t "roadSignConcept.attr.categories"}}
+              </th>
+            </c.header>
+            <c.body as |relatedRoadSignConcept|>
+              <td>
+                {{relatedRoadSignConcept.roadSignConceptCode}}
+              </td>
+              <td>
+                <img
+                  class="road-sign-image"
+                  src={{relatedRoadSignConcept.image}}
+                  alt=""
+                  role="presentation"
+                />
+              </td>
+              <td>
+                {{relatedRoadSignConcept.meaning}}
+              </td>
+              <td>
+                {{#each relatedRoadSignConcept.categories as |category|}}
+                  {{category.label}}
+                {{/each}}
+              </td>
+            </c.body>
+          </s.content>
+        </AuDataTable>
+      </div>
+    </div>
+  </div>
+  {{#if this.isToShowSubSigns}}
+    <div class="au-o-grid__item au-u-2-5@medium">
+      <div
+        class="au-c-body-container au-c-action-sidebar"
+        {{auto-focus ".au-c-action-sidebar__header"}}
+      >
+        <div class="au-o-box au-c-action-sidebar__header" tabindex="0">
+          <AuToolbar class="au-u-margin-bottom-small">
+            <AuToolbarGroup>
+              <AuHeading @level="2" @skin="3">
+                {{t "subSign.crud.edit"}}
+              </AuHeading>
+            </AuToolbarGroup>
+            <button
+              type="button"
+              class="au-c-close au-c-close--large"
+              {{on "click" this.showEditSubSigns}}
+            >
+              <AuIcon @icon="cross" @size="large" />
+              <span class="au-u-hidden-visually">
+                {{t "utility.close"}}
+              </span>
+            </button>
+          </AuToolbar>
+          <AuContent @skin="small">
+            <p>
+              {{t "subSign.crud.editText"}}
+            </p>
+          </AuContent>
+        </div>
+        <div class="au-o-box au-c-body-container au-c-body-container--scroll">
+          <div class="au-o-flow">
+            {{#each @model.roadSignConcept.subSigns as |subSign|}}
+              <div class="au-o-grid au-o-grid--tiny">
+                <div class="au-o-grid__item au-u-1-2@small au-u-1-3@medium">
+                  <AuControlCheckbox
+                    @identifier={{subSign.roadSignConceptCode}}
+                    @checked={{true}}
+                    {{on "input" (fn this.editSubSignRelation subSign)}}
```

Comment:
The latest version of appuniversum supports an `@onChange` handler. We're not on the latest version yet though.. Can you add a todo comment to make sure we tackle that in the future?